### ISSUE TITLE
Abcissas from colnames are not handled properly

### ISSUE THREAD

**Issue by rguliev** [khliland/EMSC#5]
Here is a small example:

```r
> x <- matrix(runif(50), ncol = 10)
> colnames(x) <- seq(420,600,length.out=10)
> x
           420       440        460       480        500       520        540       560        580       600
[1,] 0.1902495 0.6638507 0.96988143 0.4503034 0.02323665 0.7320788 0.44513545 0.5099727 0.00612236 0.9927394
...

> EMSC_model(x)
...
$abcissas
 [1]  1  2  3  4  5  6  7  8  9 10
 ```

I think the problem is in the line [144](https://github.com/khliland/EMSC/blob/7857921c75686db87eabd8638e53d2e2f762e33b/R/EMSC.R#L144). Because `is.ordered` is actually related to factor variables, i.e. it always returns false for colnames.

**Comment by khliland** [comment]
Dear Rustam,

You are right, this is a bug in the mentioned line. Only factor `colnames` would be caught by the `is.ordered`. I will make this more general.

Regards,
Kristian

**Comment by khliland** [comment]
Corrected in version 0.9.3.
